nemo_curator/utils/script_utils.py

```diff
@@ -395,9 +395,9 @@ def parse_distributed_classifier_args(
             required=True,
         )
         parser.add_argument(
-            "--model-path",
+            "--pretrained-model-name-or-path",
```

Comment:
Same argument name from https://github.com/NVIDIA/NeMo-Curator/pull/96.

Reply:
Thanks for this